WD-Firmware/rtos/rtosal/config/hifive-1/FreeRTOSConfig.h

```diff
@@ -0,0 +1,183 @@
+/*

```

Comment:
copy this file to nexys-a7 folder

Reply:
done https://github.com/westerndigitalcorporation/riscv-fw-infrastructure/pull/12/commits/a51f2f992c3a0ab780c6fb0d901c8f6e2b48fbf0